src/WebJobs.Script.WebHost/Middleware/Buffering/ResponseBufferingMiddlewareExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE_APACHE.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Buffering;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class ResponseBufferingMiddlewareExtensions
+    {
+        public static IApplicationBuilder UseResponseBuffering(this IApplicationBuilder builder)
```

Comment:
Are you missing some files? where is this being called?

Reply:
This extension is called in tests

Reply:
It would be nice to avoid increasing the product surface just to be able to use their tests. This is a convenience method and we can build the tests without it (or bring it into the tests instead, but it isn't required).

Reply:
Just wanted to keep tests the same as in original repo. ResponseBufferingMiddlewareExtensions was used 8 times here. Anyway removed the class.